.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 .vscode/
 .idea/
+.docx/
+Pak-Je-Kans.docx
+*.docx
```

Comment:
Ik denk dat je alleen de laatste nodig hebt. `*.docx` omvat ook `Pak-Je-Kans.docx`
`.docx/` is voor een directory genaamt `.docx`. Die heb je niet denk ik?

Het is het beste om dit netjes en minimaal te houden, om verwarring in de toekomst te vermijden.